webrtc.html

```diff
@@ -8427,113 +8429,61 @@ <h2 id="sec.stats-model">Statistics Model</h2>
-            <dt><dfn><code>packetsReceived</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span></dt>
-            <dd>
-              <p>...</p>
-            </dd>
-            <dt><dfn><code>bytesReceived</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span></dt>
-            <dd>
-              <p>...</p>
-            </dd>
-          </dl>
-        </section>
-      </div>
-      <div>
-        <pre class="idl">
-        dictionary RTCOutboundRTPStreamStats : RTCRTPStreamStats {
-             unsigned long packetsSent;
-             unsigned long bytesSent;
-};</pre>
-        <section>
-          <h2>Dictionary <a class="idlType">RTCOutboundRTPStreamStats</a>
-          Members</h2>
-          <dl data-link-for="RTCOutboundRTPStreamStats" data-dfn-for=
-          "RTCOutboundRTPStreamStats" class="dictionary-members">
-            <dt><dfn><code>packetsSent</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span></dt>
-            <dd>
-              <p>...</p>
-            </dd>
-            <dt><dfn><code>bytesSent</code></dfn> of type <span class=
-            "idlMemberType"><a>unsigned long</a></span></dt>
-            <dd>
-              <p>...</p>
-            </dd>
-          </dl>
-        </section>
-      </div>
+      <h3>Mandatory To Implement Stats</h3>
+      <p>The stats listed in [[WEBRTC-STATS]] are intended to cover a wide
+      range of use cases. Not all of them have to be implemented by every
+      WebRTC implementation.</p>
+      <p>An implementation MUST support generating statistics of the following
+      types when the corresponding objects exist on a PeerConnection, with the
+      attributes that are listed when they are valid for that object:</p>
+      <ul>
+        <li>RTCRTPStreamStats, with attributes ssrc, associateStatsId,
+        isREmote, mediaType, mediaTrackId, transportId, codecId, nackCount</li>
+        <li>RTCInboundRTPStreamStats, with all attributes from
+        RTCRTPStreamStats, and also attributes packetsReceived, bytesReceived,
+        packetsLost, jitter, packetsDiscarded</li>
+        <li>RTCOutboundRTPStreamStats, with all attributes from
+        RTCRTPStreamStats, and also attributes packetsSent, bytesSent,
+        roundTripTime</li>
+        <li>RTCPeerConnectionStats, with attributes dataChannelsOpened,
+        dataChannelsClosed</li>
+        <li>RTCDataChannelStats, with attributes label, protocol,
+        datachannelId, state, messagesSent, bytesSent, messagesReceived,
+        bytesReceived</li>
+        <li>RTCMediaStreamStats, with attributes streamIdentifer, trackIds</li>
+        <li>RTCMediaStreamTrackStats, with attributes trackIdentifier,
+        remoteSource, ended, ssrcIds, frameWidth, frameHeight, framesPerSecond,
+        framesSent, framesReceived, framesDecoded, framesDropped,
+        framesCorrupted, audioLevel</li>
```

Comment:
Added "detached" to mandatory to implement attributes.

Also filed https://github.com/w3c/webrtc-stats/issues/82 to get a better description of how "detached" is supposed to work.